demos/projects/ESPRESSIF/aziotkit/components/sample-azure-iot/Kconfig.projbuild

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+menu "Azure IoT middleware for FreeRTOS Main Task Configuration"
+
+    config AZURE_IOT_HUB_FQDN
+        string "Azure IoT Hub FQDN"
+        default "<hostname>.azure-devices.net"
+        help
+            "Set the Azure IoT Hub FQDN."
+
+    config AZURE_IOT_DEVICE_ID
+        string "Azure IoT Device ID"
+        default ""
+        help
+            "Set the Azure IoT Device ID."
+
+    config AZURE_IOT_DEVICE_SYMMETRIC_KEY

```

Comment:
>AZURE_IOT_DEVICE_SYMMETRIC_KEY [](http://example.com/codeflow?start=11&length=30)

What happen when we want to use cert based auth?

Reply:
As we spoke we will file a github issue to make the certificates a configuration in kconfig